lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -22,6 +22,49 @@
 
 LOG_MODULE_REGISTER(lte_lc_helpers, CONFIG_LTE_LINK_CONTROL_LOG_LEVEL);
 
+static int string_to_int(const char *str_buf, int base, int *output)
+{
+	int temp;
+	char *end_ptr;
+
+	errno = 0;
+	temp = strtol(str_buf, &end_ptr, base);
```

Comment:
strtoul should probably be used when cell ids can covert to numbers larger than signed ints.

Reply:
Nvm😉